orchestrator/gbt/src/default-config.toml

```diff
@@ -5,13 +5,7 @@ relayer_enabled = false
 
 # Relayer configuration options
 [relayer]
-# If we check profitability before relaying
-valset_market_enabled = false
-batch_market_enabled = true
-logic_call_market_enabled = true
-# If we relay every possible valset, only applicable when not checking
-# for profitability
-relay_every_valset = false
+## TODO update for new types
```

Comment:
When you do provide these new values, could you provide example sane values for them? E.g. `// ProfitableOnly{margin: 0.2}`  I always find it annoying when projects want you to specify numbers but never denote the units.

Reply:
that's what's currently blocking this, toml really doesn't like trying to read nested structs like this so I have to figure out how to represent it.

Reply:
Now that I've reached the calculation code I see that the margin should be 1.2 instead.

Reply:
see the example config now, maybe it should doc the values better?